src/Workspaces/Core/Portable/Options/EditorConfig/EditorConfigFileGenerator.cs

```diff
@@ -90,7 +90,7 @@ private static void AppendNamingStylePreferencesToEditorConfig(OptionSet optionS
         {
             editorconfig.AppendLine($"#### {WorkspacesResources.Naming_styles} ####");
 
-            var namingStylePreferences = optionSet.GetOption(SimplificationOptions.NamingPreferences, language);
+            var namingStylePreferences = optionSet.GetOption(SimplificationOptions.NamingPreferences, "en");
```

Comment:
❓ `language` in this context is the _programming_ language right (e.g. C# or Visual Basic)?

Reply:
Oh no